README.md

```diff
@@ -71,1043 +25,59 @@ Table of Contents
-
-```bash
-$ kogito deploy process-quarkus-example https://github.com/kiegroup/kogito-examples --context-dir=process-quarkus-example --enable-events"  
-```
-
-Or using the custom resource (CR) yaml file:
-
-```bash
-apiVersion: app.kiegroup.org/v1alpha1
-kind: KogitoApp
-metadata:
-  name: process-quarkus-example
-spec:
-  enableEvents: true
-  build:
-    envs:
-    - name: MAVEN_ARGS_APPEND
-      value: -Pevents
-    gitSource:
-      uri: https://github.com/mswiderski/kogito-quickstarts
-      contextDir: process-quarkus-example
-```
-
-The flag `--enable-events` in the CLI and the attribute `spec.enableEvents: true` in the CR tells to the operator
-to deploy a Kafka cluster in the namespace if no Kafka cluster owned by Kogito Operator is found.
-
-An [application property](#kogito-runtimes-properties-configuration) is injected into the service container for specifying the Kafka server location. 
-For Quarkus runtimes, the name of the application property is `kafka.bootstrap.servers`. 
-For Springboot, the name of the application property is `spring.kafka.bootstrap-servers`.
-
-_**Note:** The Kafka application properties are derived from the customization described in the Kogito CLI or the `KogitoApp` custom resource. Any manual changes to the properties in the ConfigMap will be overridden._
-
-Also, if the container has any environment variable with the suffix `_BOOTSTRAP_SERVERS`, they are injected by the 
-value of Kafka server location application property as well. For example, by running:
- 
-```bash
-$ kogito deploy process-quarkus-example https://github.com/kiegroup/kogito-examples --context-dir=process-quarkus-example --enable-events \
---build-env MAVEN_ARGS_APPEND="-Pevents" \
--e MP_MESSAGING_INCOMING_TRAVELLERS_BOOTSTRAP_SERVERS -e MP_MESSAGING_OUTGOING_PROCESSEDTRAVELLERS_BOOTSTRAP_SERVERS"  
-```
-The variables `MP_MESSAGING_INCOMING_TRAVELLERS_BOOTSTRAP_SERVERS` and `MP_MESSAGING_OUTGOING_PROCESSEDTRAVELLERS_BOOTSTRAP_SERVERS`
-will have the deployed Kafka service URL inject into them.
-
-### Kafka For Data Index
-
-If you do not provide a service URL to connect to Kafka or the name of a Kafka instance manually deployed via Strimzi, a new Kafka instance will be deployed with `KogitoInfra` via Strimzi.
-
-The information required to connect to Kafka will be automatically set for you by the operator to the Data Index service.
+If you have any questions, contact a Kogito Operator maintainer in the [issues page](https://github.com/kiegroup/kogito-cloud-operator/issues).
 
 ## Kogito Operator development
 
-Before you begin fixing issues or adding new features to the Kogito Operator, see [Contributing to the Kogito Operator](docs/CONTRIBUTING.MD) and [Kogito Operator architecture](docs/ARCHITECTURE.MD).
+Before you begin fixing issues or adding new features to the Kogito Operator, review the previous instructions for contributing to the Kogito Operator repository.
 
 ### Requirements
```

Comment:
Could you change this line?
```
- [Operator SDK](https://github.com/operator-framework/operator-sdk) V0.15.x or later
```
to

```
- [Operator SDK](https://github.com/operator-framework/operator-sdk) v0.15.x
```

Thanks!

Reply:
Done